src/Common/Services/CustomerEnvironmentServices.cs

```diff
@@ -59,24 +59,29 @@ public async Task<CustomerEnvironment> CreateNewEnvironmentEntityOrVersion(Custo
     /// </summary>
     /// <param name="customerEnvironment">customer environment</param>
     /// <returns></returns>
-    public async Task<CustomerEnvironment> UpdateEnvironment(CustomerEnvironment customerEnvironment, CustomerEnvironmentDeploymentPackageCollection cedpCollection)
+    public async Task<CustomerEnvironment> UpdateEnvironment(CustomerEnvironment customerEnvironment)
     {
         customerEnvironment.ChangeSet = null;
         return (await new UpdateCustomerEnvironmentInput
         {
             CustomerEnvironment = customerEnvironment,
-            DeploymentParametersMergeMode = DeploymentParametersMergeMode.Merge,
-            CustomerEnvironmentDeploymentPackageRelations = cedpCollection
+            DeploymentParametersMergeMode = DeploymentParametersMergeMode.Merge
         }.UpdateCustomerEnvironmentAsync(true)).CustomerEnvironment;
     }
 
-    public async Task<CustomerEnvironment> CreateCustomerEnvironmentForCustomerInfrastructure(CustomerEnvironment environment, string customerInfrastructureName, 
-        bool isInfrastructureAgent, CustomerEnvironmentDeploymentPackageCollection cedpCollection)
-            => (await new CreateCustomerEnvironmentForCustomerInfrastructureInput
-            {
-                CustomerInfrastructureName = customerInfrastructureName,
-                CustomerEnvironment = environment,
-                IsInfrastructureAgent = isInfrastructureAgent,
-                CustomerEnvironmentDeploymentPackageRelations = cedpCollection
-            }.CreateCustomerEnvironmentForCustomerInfrastructureAsync(true)).CustomerEnvironment;
+    public async Task<CustomerEnvironment> CreateCustomerEnvironmentForCustomerInfrastructure(CustomerEnvironment environment, string customerInfrastructureName, bool isInfrastructureAgent)
```

Comment:
Add basic documentation

Reply:
It already had docs (in the interface). Inherited doc added